src/plugins/psql_cmd_hook.py

```diff
@@ -6,13 +6,33 @@
 
 
 class PsqlCmdHook(BaseHook):
-    def __init__(self, conn_id="postgres_default", *args, **kwargs):
+    """ 
+    Excutes sql files on a postgres DB triggered by a bash cli cmd (subprocess.run) 
+    If a schema is defined in the creation of the instance, it will drop and create the schema if not exists
+    """
+    def __init__(self, db_target_schema=None, conn_id="postgres_default", *args, **kwargs):
         self.conn_id = conn_id
+        self.db_target_schema = db_target_schema
 
     def run(self, sql_files):
-        self.log.info("Running sql files: %s", sql_files)
         connection_uri = BaseHook.get_connection(self.conn_id).get_uri().split("?")[0]
         paths = " ".join(f'"{f}"' for f in sql_files)
+        
+        if self.db_target_schema:
```

Comment:
Dit is wel mooi dat schema wordt ververst maar misschien het is beter expliciet te makken via aparte method aka `refresh_schema`?

Reply:
Ja, goed idee, zal hiervoor de refresh_schema method aanmaken